### ISSUE TITLE
LiveStyle - ST3 keeps crashing

### ISSUE THREAD

**Issue by timrobinson81** [emmetio/livestyle-sublime-old#45]
LiveStyle; LOVE IT - but Sublime Text 3 keeps crashing... please advise. THANKS!

**Comment by sergeche** [comment]
Can you post crash log here and describe which actions lead to editor crashing?

**Comment by hced** [comment]
I too experience frequent hangs in ST3 (build 3047, OS X). Sorry if my report is a bit loose:

Generally, the crash occur (actually it doesn't crash ST3 but rather _hangs_ it, leaving it in unresponsive, endlessly spinning beach ball mode) when I type but a single letter, e.g. a `§` character in a normal css file that's connected with the css in the browser. Also, the "No active editor" goes on and off in somewhat odd fashion (not sure if/how this might be connected to the app hanging).

The last thing I can see in ST3's log after beach ballin' is a cut off fragment of the error message, that can't be selected and copy/pasted so here's a screenshot:

![livestyle_error](https://f.cloud.github.com/assets/93350/971064/66421ddc-05e1-11e3-8312-d1cfaf5d35b0.png)

My browser: **Chrome Canary**.
My installed plugins (each one of which I should admittedly rule out as culprits, one by one before posting this, but, well…):

Autoprefixer.sublime-package
Bootstrap 3 Snippets.sublime-package
BoundKeys
BracketHighlighter
Color Scheme - Frontend Delight
Emmet
File History
FindKeyConflicts
Fix Mac Path
JavaScript Snippets
JavaScript and jQuery API Completions
List stylesheet variables
LiveStyle
Local History
Markdown Extended
Monokai Blueberry Color Scheme
Package Control
PackageResourceViewer
PersistentRegexHighlight
Preference Helper
PyV8
SideBarEnhancements
Smart Delete
Sublime-HTMLPrettify
Sublime-JSHint
sublime-text-refactor
Summerfruit Color Scheme
Theme - Soda
eco

**Comment by sergeche** [comment]
This error means you have invalid CSS and LiveStyle can’t parse it. Check your CSS syntax for errors.

**Comment by hced** [comment]
@sergeche: Right. What happens is that ST3 hangs immediately when I enter an invalid character in the wrong place in the css file (e.g. the `§` char in the screenshot):

![livestyle_st3_hang](https://f.cloud.github.com/assets/93350/972558/486fcc96-05ff-11e3-88f3-fc7c7d9cfdd2.png)

I doubt if it's useful, but here's a gist with an excerpt from the Crash Reporter log: https://gist.github.com/hced/94c78d3e36f024b36fcc

**Comment by sergeche** [comment]
Try to update to most recent versions of ST and Chrome packages, this bug should be gone.

**Comment by hced** [comment]
My part of this issue can be "closed".

tl;dr

I installed a pristine copy of ST3 and this issue seems to be gone, which means there were problems with one of my previously installed plugins. For reference, I installed a couple of them back without interference; these work and the rest I won't reinstall:

Monokai Blueberry Color Scheme
Side​Bar​Enhancements
HTML-CSS-JS Prettify
Bracket​Highlighter
Emmet
Find​Key​Conflicts
